dali/sasha/saxref.cpp

```diff
@@ -2167,7 +2167,7 @@ class CSashaXRefServer: public ISashaServer, public Thread
     Mutex runmutex;
     bool ignorelazylost, suspendCoalescer;
     Owned<IPropertyTree> props;
-    std::unordered_map<std::string, Owned<IPropertyTree>> storagePlanes;
+    std::unordered_map<std::string, Linked<IPropertyTree>> storagePlanes;
```

Comment:
minor comment - I think you could have kept this as Owned.  It only makes a difference when the constructor is used with a pointer.